### PR TITLE
feat: setup oracle adapters deployments 

### PR DESCRIPTION
**Pull request by arturbeg**

### CODE REVIEW

File: packages/tomls/src/omnibus/reya_network.toml [hunk 1]
```diff
@@ -12,6 +12,7 @@ include = [
     "../ranks/mainnet.toml",
     "../collateral_pools/collateral_pool_1/mainnet.toml",
     "../orders_gateway/mainnet.toml",
+    "../oracle_adapters/mainnet.toml",
```
Comment (0xZenus) [inline packages/tomls/src/omnibus/reya_network.toml:15]: let's keep it out of reya_network for now and focus on reya_cronos

File: packages/tomls/src/omnibus/fuzzing.toml [hunk 2]
```diff
@@ -49,6 +50,7 @@ ranksProxyPackage = "reya-ranks:1.0.0@proxy"
 ranksRouterPackage = "reya-ranks:1.0.0@router"
 ordersGatewayProxyPackage = "reya-orders-gateway:1.0.1@proxy"
 ordersGatewayRouterPackage = "reya-orders-gateway:1.0.3@router"
+oracleAdaptersRouterPackage = "reya-oracle-adapters:1.0.0@router"
```
Comment (0xZenus) [inline packages/tomls/src/omnibus/fuzzing.toml:53]: proxyPackage is missing here

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 3]
```diff
@@ -47,6 +48,7 @@ ranksProxyPackage = "reya-ranks:1.0.0@proxy"
 ranksRouterPackage = "reya-ranks:1.0.0@router"
 ordersGatewayProxyPackage = "reya-orders-gateway:1.0.1@proxy"
 ordersGatewayRouterPackage = "reya-orders-gateway:1.0.3@router"
+oracleAdaptersRouterPackage = "reya-oracle-adapters:1.0.0@router"
```
Comment (0xZenus) [inline packages/tomls/src/omnibus/reya_cronos.toml:51]: proxyPackage is missing here

File: packages/tomls/src/omnibus/reya_network.toml [hunk 4]
```diff
@@ -41,6 +42,7 @@ exchangePassRouterPackage = "reya-exchange-pass-nft:1.0.0@router"
 ranksRouterPackage = "reya-ranks:1.0.0@router"
 ordersGatewayProxyPackage = "reya-orders-gateway:1.0.1@proxy"
 ordersGatewayRouterPackage = "reya-orders-gateway:1.0.3@router"
+oracleAdaptersRouterPackage = "reya-oracle-adapters:1.0.0@router"
```
Comment (0xZenus) [inline packages/tomls/src/omnibus/reya_network.toml:45]: proxyPackage is missing here

File: packages/tomls/src/omnibus/reya_network.toml (hunk 1 above)
Comment (arturbeg) [reply packages/tomls/src/omnibus/reya_network.toml]: https://github.com/Reya-Labs/reya-deployments/pull/54/commits/3e556ae99ff04c88cd8369b6e2c53db88f1a10e6

File: packages/tomls/src/omnibus/fuzzing.toml (hunk 2 above)
Comment (arturbeg) [reply packages/tomls/src/omnibus/fuzzing.toml:54]: done

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 5]
```diff
@@ -73,9 +76,11 @@ passivePoolProxy = "<%= imports.reyaPassivePoolProxy.contracts.InitialProxy.addr
 passivePerpProxy = "<%= imports.reyaPassivePerpProxy.contracts.InitialProxy.address %>"
 peripheryProxy = "<%= imports.reyaPeripheryProxy.contracts.InitialProxy.address %>"
 ordersGatewayProxy = "<%= imports.reyaOrdersGatewayProxy.contracts.InitialProxy.address %>"
+oracleAdaptersProxy = "<%= imports.reyaOracleAdaptersProxy.contracts.InitialProxy.address %>"
 oracleManagerProxy = "<%= imports.reyaOracleManagerProxy.contracts.InitialProxy.address %>"
 exchangePassProxy = "<%= imports.reyaExchangePassProxy.contracts.InitialProxy.address %>"
 ranksProxy = "<%= imports.reyaRanksProxy.contracts.InitialProxy.address %>"
+oracleAdaptersProxy = "<%= imports.reyaOracleAdaptersProxy.contracts.InitialProxy.address %>"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_cronos.toml:83]: duplicate

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 6]
```diff
@@ -89,6 +94,13 @@ redstonePriceFeedETH = "0x187078471869387119FB88E8F033412A0F0fdfcd"
 redstonePriceFeedBTC = "0x22Ef8eb05189cF39a347F93a33508D0ce9e37830"
 redstonePriceFeedSOL = "0x3e3762c10F018fE02701be651b5c0937C3a7d7e1"
 
+[var.stork_asset_pair_ids]
+storkAssetPairIdSOL = "SOLUSD"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_cronos.toml:98]: are these defined by stork? if not shall we add a dash between the 2 tokens?

File: packages/tomls/src/omnibus/reya_network.toml [hunk 7]
```diff
@@ -41,6 +42,8 @@ exchangePassRouterPackage = "reya-exchange-pass-nft:1.0.0@router"
 ranksRouterPackage = "reya-ranks:1.0.0@router"
 ordersGatewayProxyPackage = "reya-orders-gateway:1.0.1@proxy"
 ordersGatewayRouterPackage = "reya-orders-gateway:1.0.3@router"
+#oracleAdaptersProxyPackage = "reya-oracle-adapters:1.0.0@proxy"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_network.toml:45]: update to 1.0.1

File: packages/tomls/src/omnibus/reya_network.toml [hunk 8]
```diff
@@ -41,6 +42,8 @@ exchangePassRouterPackage = "reya-exchange-pass-nft:1.0.0@router"
...
+#oracleAdaptersRouterPackage = "reya-oracle-adapters:1.0.0@router"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_network.toml:46]: update to 1.0.1

File: packages/tomls/src/omnibus/fuzzing.toml [hunk 9]
```diff
@@ -12,6 +12,7 @@ include = [
     "../ranks/testnet.toml",
     "../collateral_pools/collateral_pool_1/testnet.toml",
     "../orders_gateway/fuzzing.toml",
+    "../oracle_adapters/fuzzing.toml",
 ]
 version = "1.0.0"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/fuzzing.toml:17]: can you please run `yarn lint:write` from root and make sure cicd is passing?

File: packages/tomls/src/omnibus/reya_network.toml (hunk 8 above)
Comment (0xZenus) [reply packages/tomls/src/omnibus/reya_network.toml]: I've actually got rid of mainnet for now, we'll push this when doing the mainnet deployment

File: packages/tomls/src/omnibus/reya_network.toml (hunk 7 above)
Comment (0xZenus) [reply packages/tomls/src/omnibus/reya_network.toml]: I've actually got rid of mainnet for now, we'll push this when doing the mainnet deployment

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 6 above)
Comment (0xZenus) [reply packages/tomls/src/omnibus/reya_cronos.toml:97]: can you ask ab ?

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 5 above)
Comment (0xZenus) [reply packages/tomls/src/omnibus/reya_cronos.toml]: got rid of it

**Review by avniculae — DISMISSED** [review]

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 6 above)
Comment (arturbeg) [reply packages/tomls/src/omnibus/reya_cronos.toml:103]: they're defined by stork

**Review by arturbeg — COMMENTED** [review]
looks good, left a few comments

**Review by 0xZenus — APPROVED** [review]

**Pull request merged**